apis/fluentbit/v1alpha2/clusterfluentbitconfig_types.go

```diff
@@ -83,7 +83,7 @@ type Service struct {
 }
 
 // +kubebuilder:object:root=true
-// +kubebuilder:resource:shortName=fbc,scope=Cluster
+// +kubebuilder:resource:shortName=cfbc,scope=Cluster
 // +genclient
```

Comment:
Why does it change to `cfbc`?

Reply:
I've set the shortName `fbc` for new namespace level `FluentBitConfig`, hence this is changed to `cfbc` to parallel `ClusterFluentdConfig` which has shortName `cfdc`